src/main/java/org/apache/sling/commons/scheduler/impl/GaugesSupport.java

```diff
@@ -227,12 +224,20 @@ public Long getValue() {
             }
         };
         logger.debug("createGauge: registering gauge : " + gaugeName);
-        this.metricRegistry.register(gaugeName, gauge);
+        try {
+            this.metricRegistry.register(gaugeName, gauge);
```

Comment:
I'd like to understand why this is needed. Ideally, the cleanup in `unregisterGauges` should be all that is needed.

Reply:
This was first solution, after this I started to look around, what is wrong. At the end I just left it in. I guess it is not needed.